### PR TITLE
Fix kafka startup script in examples

### PR DESCRIPTION
**Pull request by mstruk**
Signed-off-by: Marko Strukelj <marko.strukelj@gmail.com>

### CODE REVIEW

File: examples/docker/kafka-oauth-strimzi/kafka/start.sh [hunk 1]
```diff
@@ -14,4 +14,7 @@ wait_for_url "$URI/realms/${REALM:-demo}" "Waiting for realm '${REALM}' to be av
 
 ./simple_kafka_config.sh | tee /tmp/strimzi.properties
 
+# add extra jars to classpath
```
Comment (tombentley) [inline examples/docker/kafka-oauth-strimzi/kafka/start.sh]: Got to be worth including in the comment which extra jars this is intended to add

Comment (mstruk) [reply examples/docker/kafka-oauth-strimzi/kafka/start.sh]: Something like the following?

`# add Strimzi kafka-oauth-* jars and its dependencies to classpath`

Comment (tombentley) [reply examples/docker/kafka-oauth-strimzi/kafka/start.sh]: :+1:

Comment (mstruk) [reply examples/docker/kafka-oauth-strimzi/kafka/start.sh]: @tombentley Fixed

**Pull request merged**